<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>451</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="LinearGradientPattern">
       <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
        <gradientstop position="0.000000000000000">
         <color alpha="255">
          <red>160</red>
          <green>90</green>
          <blue>255</blue>
         </color>
        </gradientstop>
        <gradientstop position="1.000000000000000">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </gradientstop>
       </gradient>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Bitxelart</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pkb.png</normaloff>pkb.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 170, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(160, 90, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QPushButton" name="botonOriginal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>original</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonMX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>mirror x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonDM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>double mirror</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonMY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>mirror y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonSalir">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonReporte">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>reportes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAnalizar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>analizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCargar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>cargar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cBoxImg">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 255, 128);
border-color: rgb(0, 0, 0);
alternate-background-color: rgb(0, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonActualizar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>681</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Se han analizado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string> las imagenes!</string>
   </property>
  </widget>
  <widget class="QLabel" name="dimensiones">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>botonOriginal</zorder>
  <zorder>botonMX</zorder>
  <zorder>botonMY</zorder>
  <zorder>botonSalir</zorder>
  <zorder>botonReporte</zorder>
  <zorder>botonAnalizar</zorder>
  <zorder>botonCargar</zorder>
  <zorder>cBoxImg</zorder>
  <zorder>botonDM</zorder>
  <zorder>botonActualizar</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>dimensiones</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
